Update README.md #18

AstroAirafar opened the pull request:
#18  Updating badges in README.md
Added badges for forks, contributors, stars, issues etc. to make the README look more informative and elegant.
![Screenshot 2024-07-12 135643](https://github.com/user-attachments/assets/0f9dcdd4-87a5-4a1b-882b-a108214c7ddb)

AstroAirafar committed:
Update README.md

Added badges for forks, contributors, stars, issues etc. to make the README look more informative and elegant.

4darsh-Dev commented on README.md:
```diff
@@ -1,6 +1,23 @@
+</center>
+<h3><p align="center">Welcome to StudentZynergy, where teamwork makes the dream work</h3></p>
+
+![PRs Welcome](https://img.shields.io/badge/PRs-welcome-cyan.svg?style=badge&color=green&logo=openai) 
```
Do not use open ai logo , 
see the link &logo=openai

4darsh-Dev reviewed (changes requested):
@AstroAirafar Fixed the openai logo issue, 
rest is fine.

AstroAirafar committed:
Update README.md

Added the badges for forks, stars, issues, pull requests etc. to make the README look more informative and elegant

4darsh-Dev reviewed (approved):
good work
